worlds/raft/Rules.py

```diff
@@ -0,0 +1,180 @@
+from ..generic.Rules import set_rule
+from .Locations import location_table
+from .Regions import regionMap
+from ..AutoWorld import LogicMixin
+
+class RaftLogic(LogicMixin):
+    def paddleboard_mode_enabled(self, player):
```

Comment:
logic mixins are all loaded together on the collection state object, therefor its highly recommended to make custom logicMixin methods have a prefix of the game like _raft_